cli/src/pcluster/resources/compute_node/user_data.sh

```diff
@@ -39,6 +39,9 @@ fi
 Content-Type: text/cloud-config; charset=us-ascii
 MIME-Version: 1.0
 
+bootcmd:
+  - if [ "${DisableHyperThreadingManually}" = "true" ]; then for cpunum in $(cat /sys/devices/system/cpu/cpu*/topology/thread_siblings_list | tr '-' ',' | cut -s -d, -f2- | tr ',' '\n' | sort -un); do echo 0 > /sys/devices/system/cpu/cpu$cpunum/online; done; fi
```

Comment:
I'd add a comment here to explain what is the expected input and output of the cat | commands.

or add something like:
`First, it finds the total number of CPUs. Then, it loops over the second half of the CPUs. Finally, it disables each CPU in turn.`

Reply:
comment added